integration/README.md

```diff
@@ -510,10 +575,7 @@ The integration suite uses [`presets/envs.ts`](../integration/presets/envs.ts) t
 
 This is why you created the `.keys.json` file in the [initial setup](#initial-setup) step. Those secret and publishable keys are used to create environment configs. Inside GitHub actions these keys are provided through the `INTEGRATION_INSTANCE_KEYS` environment variable.
 
-Currently, we have two Clerk instances configured:
-
-- **with-email-codes**: a single session application with all toggles enabled.
-- **with-email-links**: a single session app with email links enabled and email codes disabled. Useful to test email links flows as the `<SignUp />` component currently does not support switching between email links and email codes.
+They keys defined in `.keys.json.sample` correspond with the Clerk instances in the **Integration testing** organization.
```

Comment:
This was already outdated, I don't think it's worth it to keep this in sync over time

Reply:
Correct ✅  The keys in our shared secure notes (and in GH secrets of course) are always up to date so this was not needed!